std/node/util.ts

```diff
@@ -42,60 +42,74 @@ export function inspect(object: unknown, ...opts: any): string {
   });
 }
 
+/** Checks if a value is an array. */
```

Comment:
```suggestion
/** @deprecated since v4.0.0 - use `Array.isArray()` instead. */
```

You can use `@deprecated` tag.
* JSDoc for `@deprecated` https://jsdoc.app/tags-deprecated.html
* from `@types/node` https://github.com/DefinitelyTyped/DefinitelyTyped/blob/88c6ceed5a956c081d8433f47da8713b1316f88e/types/node/util.d.ts#L26

Reply:
Thank you! I will improve JSDoc this way 👍